usl_pipeline/usl_lib/usl_lib/transformers/polygon_transformers.py

```diff
@@ -127,13 +127,15 @@ def crop_polygons_to_sub_area(
 def intersect(
     polygon_masks_to_process: Iterable[Tuple[geometry.Polygon, int]],
     multi_polygon_boundaries: geometry.MultiPolygon,
+    skip_logging: bool = False,
 ) -> Iterable[Tuple[geometry.Polygon, int]]:
     """Intersects polygons associated with integer masks with multi-polygon boundaries.
 
     Args:
         polygon_masks_to_process: Input polygons associated with integer masks that
             intersection process should be applied to.
         multi_polygon_boundaries: Multi-polygon area to apply on the input polygons.
+        skip_logging: Optional indicator that logging should be skipped.
```

Comment:
Why do we conditionally want to skip logging?

Reply:
In this particular case logging shows a lot of repeated progress of the intersection process that prevents you from seeing the rest of the logging lines. But I don't want to remove intersection progress completely though because it's useful in other cases.